src/components/Selector/Selector.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { FiArrowUpRight } from 'react-icons/fi';
+import { MassaLogo } from '../Icons/Svg/Massa/MassaLogo';
+import { Selector } from './Selector';
+
+export default {
+  title: 'Components/Selector',
+};
+
+const args = {
+  preIcon: <FiArrowUpRight />,
+  content: 'account #',
+  posIcon: <MassaLogo size={24} />,
+  amount: '0,000.00',
+};
+
+export const _Primary = {
+  render: () => <Selector {...args} customClass="!w-1/2" />,
```

Comment:
`!` is necessary ?

Reply:
Mystery: for some class yes, for others no! With `w-1/3` no, with `w-1/2` yes... Anyway I left there to emphasise that if it does not take in account the !(**!important**) will make it.